### PR TITLE
Revising Pulse Project component & adding options

### PR DESCRIPTION
**Pull request by gvn**
Related issue: #1223

For review see: https://foundation-mofostaging-pr-1226.herokuapp.com/style-guide/

### CODE REVIEW

**Comment by kristinashu** [comment]
Layout looks good @gvn! Just confirming that Global Sprint page will still show all the projects and in reverse order?

**Comment by gvn** [comment]
Yep, I have it set to reverse chronological order by default. 

Just demo-ing that you can change it here in the style guide...

File: source/pug/views/style-guide.pug [hunk 1]
```diff
@@ -76,8 +76,8 @@ block content
         h4 .join-us.full-page
         .join-us.full-page
 
-        h4 .pulse-project-list(for="QUERY")
-        .pulse-project-list(for=`mozfest`)
+        h5 .pulse-project-list(for="SEARCH_QUERY", size="MAX_TO_DISPLAY", rev="BOOLEAN_TO_INDICATE_ORDERING")
```
Comment (mmmavis) [inline source/pug/views/style-guide.pug]: this should stay as `h4`?

Comment (gvn) [reply source/pug/views/style-guide.pug]: Doesn't really matter...just the style guide. It was kinda large as a H4.

**Pull request merged**